sql/common/src/test/scala/org/apache/sedona/sql/rasteralgebraTest.scala

```diff
@@ -612,6 +612,18 @@ class rasteralgebraTest extends TestBaseScala with BeforeAndAfter with GivenWhen
 
     }
 
+    it("Passed RS_Count with raster") {
+      var df = sparkSession.read.format("binaryFile").load(resourceFolder + "raster/raster_with_no_data/test5.tiff")
+      df = df.selectExpr("RS_FromGeoTiff(content) as raster")
+      var actual = df.selectExpr("RS_Count(raster, 1, true)").first().getInt(0)
+      var expected = 928192
```

Comment:
pls test all possible flavors of this function in scala as well to ensure proper porting